lptree.pas

```diff
@@ -4457,8 +4499,16 @@ function TLapeTree_Operator.Compile(var Offset: Integer): TResVar;
       finally
         Free();
       end;
-    except on E: lpException do
-      LapeException(lpString(E.Message), DocPos);
+    except
+      on E: lpException do
```

Comment:
This could probably also be done by setting a boolean flag in EvalRes, when we find out it should be done by operator overload.